w06-ruby/d1-dawn-peer-review/README.md

```diff
@@ -0,0 +1,44 @@
+# Project 1
+![request](imgs/request.gif)
+## Peer Review
+
+| Objectives |
+| :--- |
+| Create a Github Issue for your project |
+| Fork & then clone your partner's project |
+| Pair program to resolve the Github Issue & submit a Pull Request |
+
+**Please keep track of time -- you should spend roughly an hour on each project**
+
+###1. Create an Issue (10m)
+
+* Discuss what your application does at a high level. This should be one or two sentences, an elevator pitch. (5m)
+* Discuss 1 minor issue in your project. Make sure it's a simple issue, like a small addition, that could be solved relatively quickly with the tools we already have - jQuery, express, etc.
+* On Github, create a new Issue. Be descriptive of what is wrong and how you'd like it fixed. Maybe give some rough steps you'd take to solve it.
+
+![issues](imgs/issue.png)
+
+###2. Setup the Project (30m)
+1. __Fork__ your partner's project.
+2. __Clone__ the fork you just made locally.
+3. Make a new __branch__ named after the issue you're solving.
+    - ` git checkout -b fix_signin_redirect`
```

Comment:
This is the first time that a good portion of the students have seen this command (or used a branch other than master).  Should we provide more info?